### ISSUE TITLE
per_page is not honored

### ISSUE THREAD

**Issue by zeke** [librariesio/libraries.io#753]
Setting a `per_page` query param, but the response always contains 30 results.

```sh
npm i -g json
curl -s "https://libraries.io/api/npm/electron-prebuilt/dependents/?page=4&per_page=100" | json -a name | wc -l

# 30
```

cc @jlord

**Comment by andrew** [comment]
Might want to remove your api key ;)

**Comment by zeke** [comment]
Done! 🔒

**Comment by andrew** [comment]
Believe I have fixed this for you, can you confirm?

**Comment by zeke** [comment]
Still returning 30:

```
❯ curl -s "https://libraries.io/api/npm/electron-prebuilt/dependents?page=1&per_page=100" | json -a name | wc -l
      30
```

**Comment by andrew** [comment]
This one should be fixed now, still working #744

**Comment by zeke** [comment]
Fixed! Thanks.
